src/runtime/styled.js

```diff
@@ -9,6 +9,12 @@ const camelCase = str =>
     '',
   );
 
+function resolveVariants(variants) {
+  return variants.length
+    ? variants.reduce((c, n) => (n ? `${c} ${n}` : c), '')
```

Comment:
isn't this just `.filter(Boolean).join(' ')`?

Reply:
true!